thirdeye-ui/src/app/components/anomalies-create/create-anomaly-properties-form/create-anomaly-properties-form.component.tsx

```diff
@@ -0,0 +1,249 @@
+                                            }
+                                        )
+                                    )}
+                                    size="small"
+                                    variant="outlined"
+                                />
+                            )}
+                            value={formFields.alert || undefined}
+                            onChange={(_, selectedValue) => {
+                                selectedValue &&
+                                    handleSetField("alert", selectedValue);
+                            }}
+                        />
+                    }
+                    labelComponent={
+                        <>
+                            <Typography variant="body2">
+                                {formLabels.alert}
+                            </Typography>
+                            {!!formFields.alert &&
+                                linkRendererV1(
+                                    t("label.view-entity", {
+                                        entity: t("label.alert"),
+                                    }),
+                                    getAlertsAlertViewPath(formFields.alert.id),
+                                    false,
+                                    undefined,
+                                    true,
+                                    "_blank"
+                                )}
+                        </>
+                    }
+                />
+
+                {/* Since these fields are read-only, only show them when the alert is loaded */}
+                {!!formFields.alert &&
+                    (
+                        Object.keys(
+                            readOnlyFormFields
+                        ) as (keyof CreateAnomalyReadOnlyFormFields)[]
+                    ).map(
+                        (
+                            readOnlyKey: keyof CreateAnomalyReadOnlyFormFields
+                        ) => (
+                            <InputSection
+                                inputComponent={
+                                    <TextField
+                                        fullWidth
+                                        required
+                                        InputProps={{
+                                            margin: "none",
+                                            readOnly: true,
+                                        }}
+                                        name={readOnlyKey}
+                                        title="These properties are derived from the selected alert"
+                                        type="string"
+                                        value={
+                                            readOnlyFormFields[readOnlyKey] ||
+                                            ""
+                                        }
+                                        variant="outlined"
+                                    />
```

Comment:
We should add the `disable` so it has the disable look or hide it. Im actually in favor of hiding. The alert probably will suffice. Or we can provide a caption so that the text is tiny to provide these details. I don't think it adds value to display dataset and datasource

Reply:
`disabled` would look better as the fields would be greyed out, but these fields might just be removed from the form (as discussed). 

This comment can be resolved once there is confirmation about if these fields can indeed be removed, or should they be retained.

Reply:
Removed these fields altogether